jenkins-pipelines/operator/upgrade/upgrade-platform-k8s-eks.jenkinsfile

```diff
@@ -4,8 +4,13 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 rollingOperatorUpgradePipeline(
     backend: 'k8s-eks',
-    region: 'eu-north-1',
+    region: 'us-west-2',
     test_name: 'upgrade_test.UpgradeTest.test_kubernetes_platform_upgrade',
     test_config: 'test-cases/scylla-operator/kubernetes-platform-upgrade.yaml',
     availability_zone: 'a,b',
+    base_versions: '["2024.1.21"]',
+    new_version: '["2024.1.21"]',
```

Comment:
`new_version` is identical to base one and having a K8S upgrade, this condig option is not needed here.

Reply:
removed from both files